syntax/verilog_systemverilog.vim

```diff
@@ -157,7 +156,7 @@ endif
 if index(s:verilog_syntax_fold, "class") >= 0 || index(s:verilog_syntax_fold, "all") >= 0
     syn region  verilogFold
         \ matchgroup=verilogStatement
-        \ start="\(typedef\s\+\)\@<!\<class\>"
+        \ start="\(typedef\s\+\)\@<!\<\(interface\s\+\)\?class\>"
```

Comment:
Yes. Otherwise the interface keyword in an interface class doesn't get syntax highlighted. I couldn't think of any other way to achieve this.